src/codemodder/codemods/transformations/remove_empty_string_concatenation.py

```diff
@@ -41,20 +50,21 @@ def handle_node(
     ) -> cst.BaseExpression:
         left = updated_node.left
         right = updated_node.right
-        if self._is_empty_string_literal(left):
-            if self._is_empty_string_literal(right):
+        if _is_empty_string_literal(left):
+            if _is_empty_string_literal(right):
                 return cst.SimpleString(value='""')
             return right
-        if self._is_empty_string_literal(right):
-            if self._is_empty_string_literal(left):
+        if _is_empty_string_literal(right):
+            if _is_empty_string_literal(left):
                 return cst.SimpleString(value='""')
             return left
         return updated_node
 
-    def _is_empty_string_literal(self, node):
-        match node:
-            case cst.SimpleString() if node.raw_value == "":
-                return True
-            case cst.FormattedString() if not node.parts:
-                return True
-        return False
+
+def _is_empty_string_literal(node):
```

Comment:
same function as the other file so move to a utility module and remove the leading `_` since it won't be private